docs/developer/tutorials/snmp/profile-format.md

```diff
@@ -309,6 +309,36 @@ metrics:
         index: 1
 ```
 
+##### Mapping index to tag string value
+
+You can use the following syntax to map indexes to tag string values.
+In the example below, the submitted metrics will be `snmp.ipSystemStatsHCInReceives` with tags like `ipversion:ipv6`.
+
+
+```yaml
+metrics:
+- MIB: IP-MIB
+  table:
+    OID: 1.3.6.1.2.1.4.31.1
+    name: ipSystemStatsTable
+  forced_type: monotonic_count
+  symbols:
+  - OID: 1.3.6.1.2.1.4.31.1.1.4
+    name: ipSystemStatsHCInReceives
+  metric_tags:
+  - index: 1
```

Comment:
can you add that `index` is the column within the same table, or link to the index section in tagging? I think the index doc needs to be updated too https://datadoghq.dev/integrations-core/tutorials/snmp/profile-format/#using-an-index

According to what we discussed, `index` is the column within the same table, not the index in `INDEXES` list in MIB definition

Reply:
Updated here: https://github.com/DataDog/integrations-core/pull/8063/commits/0e6384eac2d0b3673651178ad28d21f6ca7d5290